azure-pipelines/code-coverage/common.yml

```diff
@@ -0,0 +1,46 @@
+# File: azure-pipelines\code-coverage\common.yml
+# Description: Code coverage for common
+# Variable: 'mvnAccessToken' was defined in the Variables tab
+# https://docs.microsoft.com/en-us/azure/devops/organizations/accounts/use-personal-access-tokens-to-authenticate
+# Variable: 'commonBranchName' common branch to be used when running the pipeline manually
+name: $(date:yyyyMMdd)$(rev:.r)
+
+trigger: none
```

Comment:
Do we want both (GitHub Action and Azure ) pipelines to run on PR?

Reply:
Not at the moment - this will be done in the near future when we have a nightly run